#include "ros/ros.h"
#include "std_msgs/String.h"
#include <stdlib.h>
#include "rcnn_live_detector/Prediction.h"
#include "rcnn_live_detector/PredictionsList.h"
#include "rcnn_live_detector/paquete_imagenes.h"
#include "rcnn_live_detector/imageTagger.h"
#include "rcnn_live_detector/profundidadServer.h"
#include "rcnn_live_detector/projectPandaProfundidadServer.h"
#include "rcnn_live_detector/msgTomarObjeto.h"
#include <iostream>
#include <fstream>
#include <sstream>
#include "ros/package.h"
#include <geometry_msgs/Point.h>
#include <cv_bridge/cv_bridge.h>
#include <opencv2/opencv.hpp>
#include <sensor_msgs/Image.h>
#include <sensor_msgs/PointCloud2.h>
#include <limits>
#include <image_transport/image_transport.h>
#include <opencv/cv.h>
#include <image_geometry/pinhole_camera_model.h>
#include <tf/transform_listener.h>
#include <boost/foreach.hpp>
#include <sensor_msgs/image_encodings.h>
#include <string>
#include <math.h> 
#include <stdio.h>  
#include <cmath>
#include <tf/transform_datatypes.h>
#include "naoqi_bridge_msgs/JointAnglesWithSpeed.h"
#include "nav_msgs/Odometry.h"
#include "std_msgs/String.h"
#include <tf/transform_datatypes.h>
#include "std_msgs/String.h"


#include "geometry_msgs/PoseStamped.h"


#include <vector>
#include <math.h>
#include <ros/console.h>
#include <map>
#include "std_msgs/String.h"
#include <boost/foreach.hpp>
//#include <array>

#include <unistd.h>//para medir tiempo delay



ros::Subscriber target_sub;
boost::shared_ptr<ros::NodeHandle> nh_ptr_;


void callBack(const std_msgs::String::ConstPtr& messageObject)
{
	
}


int main(int argc, char **argv)
{
  

  ros::init(argc, argv, "SearchTarget");
  ROS_INFO("inicio de nodo SearchTarget");
  nh_ptr_ = boost::make_shared<ros::NodeHandle>();

  target_sub = nh_ptr_->subscribe<std_msgs::String>("/speechclassObject", 1000, callBack);


  ros::spin();

  return 0;

}